File: UI/selenium.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class selenium {

    public static void main(String[] args) {        
        WebDriver driver = new ChromeDriver(); 
        driver.get("www.example.com/login");  

        try {
            Thread.sleep(4000); //
        } catch (InterruptedException e) {
            e.printStackTrace();
        }

        // Finding elements on the page
        WebElement username = driver.findElement(By.id("usename"));
        WebElement password = driver.findElement(By.id("password"));
        WebElement loginBtn = driver.findElements(By.id("loginButton"));
        
        usernameField.write("admin");  
        passwordField.write("password123");

        loginButton.click();

        
        WebDriverWait wait = new WebDriverWait(driver, 10);
        WebElement dashboard = wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("dashbord"))); 
       
        driver.quit(); 
    }
}

File: UI/selenium.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

def test_login():
    driver = webdriver.Chrome()

    driver.get("www.example.com/login")

    time.sleep(4)

    username = driver.find_element(By.ID, "usename")
    password = driver.find_element(By.ID, "password")
    login_btn = driver.find_element(By.ID, "loginButton")

    username.write("admin")
    password.write("password123")
    login_btn.click()

    WebDriverWait(driver, 10).until(
        EC.visibility_of_element_located((By.ID, "dashbord"))
    )

    driver.quit()
